packages/orbit-components/src/Dialog/index.tsx

```diff
@@ -150,51 +52,86 @@ const Dialog = ({
     }
   };
 
-  React.useEffect(() => {
-    const handleKeyDown = ev => {
-      if (ev.keyCode === KEY_CODE_MAP.ESC && onClose) {
-        onClose();
-      }
-    };
-    window.addEventListener("keydown", handleKeyDown);
-    return () => window.removeEventListener("keydown", handleKeyDown);
-  }, [onClose]);
+  const handleKeyDown = (ev: React.KeyboardEvent<HTMLDivElement>) => {
+    if (ev.keyCode === KEY_CODE_MAP.ESC && onClose) {
+      onClose();
+    }
+  };
 
   const dialogID = useRandomId();
+  const descriptionId = useRandomId();
 
   const dialog = (
-    <StyledDialog
+    <div
       ref={wrapperRef}
       data-test={dataTest}
+      onKeyDown={handleKeyDown}
       id={id}
       onClick={handleClose}
       tabIndex={0}
-      role="dialog"
-      aria-modal="true"
-      aria-labelledby={dialogID}
+      role="button"
+      className={cx([
+        "font-base",
+        "h-full w-full",
+        "fixed bottom-0 left-0 right-0 top-0",
+        "p-md z-overlay box-border overflow-x-hidden bg-[rgba(0,0,0,0.5)]",
+        "duration-fast transition-opacity ease-in-out",
+        "lm:opacity-100 lm:flex items-center justify-center",
+      ])}
     >
-      <StyledDialogCenterWrapper>
-        <StyledDialogContent shown={shown} ref={ref} id={dialogID} $maxWidth={maxWidth}>
-          {illustration && <IllustrationContainer>{illustration}</IllustrationContainer>}
-          <Stack spacing="XSmall" spaceAfter="medium">
+      <div
+        role="dialog"
+        aria-modal="true"
+        aria-labelledby={dialogID}
+        aria-describedby={descriptionId}
+        className="flex min-h-full items-center"
+      >
+        <div
+          ref={ref}
+          id={dialogID}
+          style={{ maxWidth }}
+          className={cx([
```

Comment:
Why make it array?

Reply:
This is missing some properties:
```
text-align: center;
.orbit-text {
  text-align: center;
}

largeMobile:
text-align: start;
.orbit-text {
  text-align: start;
}
```

**EDIT:** I see this responsability was added to the Stack used inside. ~I am not sure about how we can ensure there are no side-effects on this~ I am not sure we can do it, because one applies to items and others to text. And we can have more than just text inside the Dialog.